### ISSUE TITLE
Unknown rule named-grid-areas-no-invalid

### ISSUE THREAD

**Issue by markgoho** [stylelint/stylelint-config-standard#174]
> Clearly describe the bug

`Unknown rule named-grid-areas-no-invalid`

> What steps are needed to reproduce the bug?

1. Update to stylelint 13.12.0, update stylelint-config-standard to 21.0.0
2. See stylelint complain about unknown rule.

> What did you expect to happen?

For stylelint to be aware of the new rule

> What actually happened (e.g. what warnings or errors did you get)?

An error in the IDE saying it doesn't know about the rule

Additional information:

If I comment out the extends `- stylelint-config-standard` stylelint stops complaining about the issue.

**Comment by hudochenkov** [comment]
Most likely your IDE uses different version that you've installed. Do you have the same problem when running stylelint via CLI?

**Comment by jeddy3** [comment]
This usually happens if there's a globally installed version of stylelint that hasn't been updated.

Run:

```
npm uninstall -g stylelint
npm install -D stylelint@latest stylelint-config-standard@latest
```

And, as @hudochenkov said, you may need to update your IDE extension.

Feel free to reopen if these don't feel the issue for you.

**Comment by markgoho** [comment]
Thanks all for the help troubleshooting. No errors from the CLI -- this is definitely an issue with my IDE integration.

**Comment by snood1205** [comment]
I have recently been experiencing this issue as well, but it appears on CI, which does a fresh install of everything each time it runs. Is there a way to just ignore the rule `named-grid-areas-no-invalid` so it doesn't cause an error? This is hapening upon upgrade from 20.0.0 to 21.0.0. The error message is
```
 1:1  ✖  Unknown rule named-grid-areas-no-invalid  named-grid-areas-no-invalid
  stderr: Nothing written
```

**Comment by jeddy3** [comment]
> is there a way to just ignore the rule named-grid-areas-no-invalid so it doesn't cause an error? 

Yes.

```json
{
  "extends": ["stylelint-config-standard"],
  "rules": {
    "named-grid-areas-no-invalid": null
  }
}
```

But you shouldn't have to do this.

> I have recently been experiencing this issue as well, but it appears on CI,

It would be good to get to the bottom of this. 

What output do you get when you run these two commands in your project root:

```
stylelint --version
```

```
npx stylelint --version
```

**Comment by snood1205** [comment]
@jeddy3 Here are the results of it from my terminal. 
```
$ stylelint
--version
zsh: command not found: stylelint
$ npx stylelint --version
13.12.0
$ yarn stylelint --version
yarn run v1.22.5
$ [Project Root Path]/node_modules/.bin/stylelint --version
13.12.0
✨  Done in 0.35s.
```
All of those results were expected as I never installed stylelint globally.

**Comment by beetaa** [comment]
> > is there a way to just ignore the rule named-grid-areas-no-invalid so it doesn't cause an error?
> 
> Yes.
> 
> ```json
> {
>   "extends": ["stylelint-config-standard"],
>   "rules": {
>     "named-grid-areas-no-invalid": null
>   }
> }
> ```
> 
> But you shouldn't have to do this.
> 
> > I have recently been experiencing this issue as well, but it appears on CI,
> 
> It would be good to get to the bottom of this.
> 
> What output do you get when you run these two commands in your project root:
> 
> ```
> stylelint --version
> ```
> 
> ```
> npx stylelint --version
> ```

both the stylelint and configuration package are latest versions, the vscode extension is also the latest version, 0.86.0, but still not work after modified the **rules** part.

**Comment by beetaa** [comment]
I think the problem comes from commit 554e2987b584ccebfc34b362e9c1e8072276d4c7

**Comment by beetaa** [comment]
@markgoho @jeddy3 I just figured it out by specifying vscode extenstion's setting key `"stylelint.stylelintPath": "{path of stylelint module}"`, for example, I installed the stylelint globally with pnpm, so the path should be like `/usr/pnpm-global/4/node_modules/stylelint`.

now I think the problem is basically because of the confliction between editor extension's stylelint version and the installed package's stylelint version.

hope this helps.
